support/bazel/README.md

```diff
@@ -0,0 +1,85 @@
+# Bazel support
+
+Bazel is an open-source build tool.
+More information about Bazel can be found [here](https://bazel.build/).
+
+## Using the fmt repository with Bazel
+
+Even the `fmt` repository does not contain a `WORKSPACE` file in its root directory,
```

Comment:
Seems like "though" is missing after "Even".

Reply:
Added "though"